### ISSUE TITLE
can you use camp_zipnerf with webbased smerf viewer?

### ISSUE THREAD

**Issue by comsyspro** [jonbarron/camp_zipnerf#8]
is it possible to create a webbased walkthrough in combination with camp_zipnerf and smerf-3d?

smerf-3d github project, webbased smerf viewer:
https://github.com/smerf-3d/smerf-3d.github.io

if yes, how to use the content and include it into the smerf viewer? are there any tutorials?

**Comment by jonbarron** [comment]
No, this isn't something you can do easily with the existing code we've open-sourced.

**Comment by comsyspro** [comment]
do you know how you can achieve that and are you connected to the people form smerf-3d project or is that another developer group?
can you say how this is done and is your project a post step for creating these webbased viewers?
https://smerf-3d.github.io/#demos
